contracts/JBTokenStore.sol

```diff
@@ -232,32 +231,27 @@ contract JBTokenStore is JBControllerUtility, JBOperatable, IJBTokenStore {
         // The total supply can't exceed the maximum value storable in a uint224.
         if (totalSupplyOf(_projectId) > type(uint224).max) revert OVERFLOW_ALERT();
```

Comment:
IIC, OZ erv20vote is now max uint208

https://github.com/OpenZeppelin/openzeppelin-contracts/blob/0950532d9a89ac652dbbb2aed56678bdfd624c21/contracts/token/ERC20/extensions/ERC20Votes.sol#L30

Reply:
(but OZ catches it in _update, do we need to do it too?)